utils/src/treev2/quickPickWizard/quickPickAzureResource/QuickPickGroupStep.ts

```diff
@@ -16,21 +17,24 @@ interface GroupQuickPickOptions extends SkipIfOneQuickPickOptions {
 
 export class QuickPickGroupStep extends GenericQuickPickStep<AzureResourceQuickPickWizardContext, GroupQuickPickOptions> {
     public constructor(tdp: vscode.TreeDataProvider<unknown>, options: GroupQuickPickOptions) {
-        super(
-            tdp,
-            {
-                ...options,
-                skipIfOne: true, // Group is always skip-if-one
-            }
-        );
+        super(tdp, {
+            ...options,
+            skipIfOne: true, // Group is always skip-if-one
+        });
     }
 
-    protected isDirectPick(_node: vscode.TreeItem): boolean {
+    readonly pickFilter: PickFilter = new GroupPickFilter(this.pickOptions);

```

Comment:
Since this isn't being called within the constructor, I'm not 100% sure that `this.pickOptions` will be available. I don't know the order for sure but I thought that class-scoped initializers like this get called _before_ the constructor.